wallet/src/bin/web_server.rs

```diff
@@ -328,9 +330,10 @@ mod tests {
         println!("mnemonic: {}", mnemonic);
         let password = "my-password";
 
+        env::set_var("PATH_STORAGE", "/tmp");
```

Comment:
Slightly better to use
```suggestion
        let dir = TempDir::new("wallet_last_used_test").unwrap();
        env::set_var("PATH_STORAGE", &format!("{:?}", dir));
```
but I won't let that block the review